src/crypto/index.js

```diff
@@ -246,6 +246,18 @@ Crypto.prototype.uploadDeviceKeys = function() {
     });
 };
 
+/**
+ * Stores the current key count which will be handled postponed (in a call of
+ * _onSyncCompleted)
+ *
+ * @param {Number} currentKeyCount to be stored and handled later
+ */
+Crypto.prototype.updateCurrentKeyCount = function(currentKeyCount) {
```

Comment:
Can we rename this (and update the comment) to reflect the fact that these are *one-time* keys? `updateOneTimeKeyCount` or sth